apps/nextjs-website/src/app/profile/personal-data/page.tsx

```diff
@@ -57,6 +75,29 @@ const PersonalData = () => {
       <InfoCard
         cardTitle={t('personalData.dataSection')}
         items={dataSectionItems}
+        onValue={(items: InfoCardItemProps[]) => {
+          const oldItems = [...dataSectionItems];
+
+          if (!user) return;
+
+          setUserAttributes(
+            {
+              ...user.attributes,
+              given_name: items[0].value || '',
```

Comment:
Maybe you could replace this and the following `|| ''` with something like this:
```suggestion
              ...(items[0].value && { given_name: items[0].value }),
```